examples/app_commands_and_api/command.py

```diff
@@ -12,6 +12,6 @@ class CustomConfig(BaseModel):
 class CustomCommand(ClientCommand):
     def run(self):
         parser = ArgumentParser()
-        parser.add_argument("--name", type=str)
+        parser.add_argument("--name", type=str, required=True)
```

Comment:
Why are we making name a required argument?

Reply:
Removed.